package.json

```diff
@@ -111,7 +111,7 @@
   },
   "peerDependencies": {
     "@folio/service-interaction": "^3.1.0",
-    "@folio/stripes": "^9.2.0",
+    "@folio/stripes": "^10.0.0",
```

Comment:
Bump service-interaction to v4. That's available in CI and so is OK to merge now. You won't be able to _release_ this version until service-interaction v4 is, like stripes v10, formally released.